openquake/hazardlib/tests/geo/surface/gridded_test.py

```diff
@@ -70,10 +71,17 @@ def test_get_ry0_distance(self):
                           self.mesh)
 
     def test_get_strike(self):
-        np.testing.assert_equal(np.nan, self.surf.get_strike())
+        strike = self.surf.get_strike()
+        strike_2 = self.surf2.get_strike()
+        strike_3 = self.surf3.get_strike()
+        self.assertEqual(strike, 0.707115755836035)
```

Comment:
Can you please add a note explaining how you defined/computed the expected values?

Reply:
incorporated